sor/src/main/java/com/bazaarvoice/emodb/sor/db/astyanax/StitchedRow.java

```diff
@@ -7,17 +7,19 @@
 
 public class StitchedRow extends AbstractGettableData implements Row {
 
-    private ByteBuffer _content;
-    private Row _oldRow;
-    private int _contentIndex;
-    private CodecRegistry _codecRegistry;
+    private final ByteBuffer _content;
+    private final Row _oldRow;
+    private final int _contentIndex;
+    private final CodecRegistry _codecRegistry;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this field "_codecRegistry" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.qa.bazaarvoice.com/coding_rules#rule_key=squid%3AS00116)